pkg/config/configobserver/configobserver_test.go

```diff
@@ -40,6 +41,50 @@ func Test_ConfigObserver_ChangeSupportConfig(t *testing.T) {
 			},
 			expErr: fmt.Errorf("insights secret interval must be a duration (1h, 10m) greater than or equal to ten seconds: too short"),
 		},
+		{name: "interval incorrect format",
+			config: map[string]*corev1.Secret{
+				pullSecretKey: {Data: map[string][]byte{
+					".dockerconfigjson": nil,
+				}},
+				supportKey: {Data: map[string][]byte{
+					"interval": []byte("every second"),
+				}},
+			},
+			expErr: fmt.Errorf("insights secret interval must be a duration (1h, 10m) greater than or equal to ten seconds: time: invalid duration \"every second\""),
+		},
+		{name: "reportPullingDelay incorrect format",
+			config: map[string]*corev1.Secret{
+				pullSecretKey: {Data: map[string][]byte{
+					".dockerconfigjson": nil,
+				}},
+				supportKey: {Data: map[string][]byte{
+					"reportPullingDelay": []byte("every second"),
+				}},
+			},
+			expConfig: &config.Controller{}, // it only produces a warning in the log
+		},
+		{name: "reportMinRetryTime incorrect format",
+			config: map[string]*corev1.Secret{
+				pullSecretKey: {Data: map[string][]byte{
+					".dockerconfigjson": nil,
+				}},
+				supportKey: {Data: map[string][]byte{
+					"reportMinRetryTime": []byte("every second"),
+				}},
+			},
+			expConfig: &config.Controller{}, // it only produces a warning in the log
+		},
+		{name: "reportPullingDelay incorrect format",
+			config: map[string]*corev1.Secret{
+				pullSecretKey: {Data: map[string][]byte{
+					".dockerconfigjson": nil,
+				}},
+				supportKey: {Data: map[string][]byte{
+					"reportPullingDelay": []byte("every second"),
+				}},
+			},
+			expConfig: &config.Controller{}, // it only produces a warning in the log
+		},
```

Comment:
It looks like we have this one twice.

Reply:
Should it be for `reportPullingTimeout` ?

Reply:
yep, nice catch :)